glslc/README.asciidoc

```diff
@@ -252,22 +252,30 @@ code, such as identifier names and line numbers.
 NOTE: Currently this option has no effect.  Full functionality depends on
 glslang support for generating debug info.
 
-==== `-masm=<dialect>`
-
-Select dialect for compilation output. Supported options are listed in
-<<output-dialect-selection,dialect selection table>>
-[[output-dialect-selection]]
-.Output Dialect Selection
+==== `-mfmt=<format>`
+
+Select output format for compilation output in SPIR-V binary code form.
+Supported options are listed in <<binary-output-format-selection,binary output
+format selection table>>. This option is only valid to be used when the
+compilation output is SPIR-V binary code. Specifying any options listed below
+when the output is not SPIR-V binary code, like disassembly (with `-S`
+specified), text (with `-M`, `-MM` or `-E` specified) will trigger an error.
+[[binary-output-format-selection]]
+.Binary Output Format Selections
 |===
-|Dialect option |Description
-
-|c-init-list    |Output SPIRV binary code as a text file containing C-style +
-                 initializer list. This option is valid when the compilation +
-                 output is SPIRV binary code. It is not valid to use when the +
-                 output is not SPIV binary code, like when `-S`, `-E`, `-M`, +
-                 `-MM` are turned on. +
-
-                 Example: `glslc -c main.vert -o output_file.txt` +
+|Format option  |Description
+
+|bin            |Output SPIR-V binary code as it is.
```

Comment:
"as it is" -> "as a sequence of binary 32-bitwords in host native endianness.  This is the default."